app/controllers/claims_controller.rb

```diff
@@ -10,6 +10,8 @@ class ClaimsController < BasePublicController
   before_action :check_claim_not_in_progress, only: [:new]
   before_action :clear_claim_session, only: [:new]
   before_action :prepend_view_path_for_policy
+  before_action :set_teacher_details, only: [:teacher_detail]
```

Comment:
Isn't better if we can do something like this `only: :teacher_detail` as we are restricting this to only one action. Its better to not use array

Reply:
Please replace `teacher_detail` to `teacher_details`

Reply:
Since the "teacher_detail" action was empty, I did not include it here.